inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/config/filebased/ConfigurationDirectoriesWatcher.java

```diff
@@ -24,8 +24,7 @@
  */
 @Service
 @Slf4j
-@ActivationConfigCondition("config.fileBased.watch")
-public class ConfigurationDirectoriesWatcher {
+public class ConfigurationDirectoriesWatcher extends DynamicallyActivatableService {
```

Comment:
should we keep only one? do we have any benefit in having to implementations, or better said do we see any situation where the polling approach is not sufficient?